### PR TITLE
Add new template to AllTests & change template to Entity string generate 

### PR DESCRIPTION
**Pull request by ditansu**
To resolve issue #17 
1. Changed the AutoModelGeneratable.stencil to Entity's string generate as extension to Models's classes 
2. Updated all models - removed Entity string  from Models' classes  as duplicated 
3. Added  AllTests generate to support Linux tests -- it's important for CI tests stage on Linux

### CODE REVIEW

**Comment by ditansu** [comment]
There are two wrong files in commit: Routes.swift and Config+Setup.swift

File: Templates/AutoModelGeneratable.stencil [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {# Defining macros for each annotation #}
 
 
+{% macro AddEntity type %}
```
Comment (dannydaddy3) [inline Templates/AutoModelGeneratable.stencil]: Remove this newline under extension. To avoid having 2 newlines after imports as it's now

**Review by dannydaddy3 — CHANGES_REQUESTED** [review]

File: Templates/AutoModelGeneratable.stencil (hunk 1 above)
Comment (dannydaddy3) [reply Templates/AutoModelGeneratable.stencil]: And regenerate models)

File: Sources/App/Routes/Routes.swift [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
+      try json.set("hello", "world")
+      return json
+    }
+    
+    get("plaintext") { req in
+      return "Hello, world!"
     }
+    
+    let heartbeat = HeartbeatController()
+    resource("heartbeat",heartbeat)
```
Comment (dannydaddy3) [inline Sources/App/Routes/Routes.swift]: I think you accidentally branched from heartbeat branch, need to remove this

File: Templates/LinuxMainAllTestsGenerate.stencil [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// sourcery:file:Tests/LinuxMain.swift
+#if os(Linux)
+import XCTest
+
+{% for type in types.classes|based:"TestCase" %}
+{% if not type.annotations.disableTests %}extension {{ type.name }} {
```
Comment (dannydaddy3) [inline Templates/LinuxMainAllTestsGenerate.stencil:6]: Add newline under `for type in types...`, and remove newline under `import XCTest`.

Thus extension won't be glued to each other

**Review by dannydaddy3 — CHANGES_REQUESTED** [review]

**Review by dannydaddy3 — APPROVED** [review]

File: Templates/LinuxMainAllTestsGenerate.stencil (hunk 3 above)
Comment (ditansu) [reply Templates/LinuxMainAllTestsGenerate.stencil:6]: fixed

**Review by petertretyakov — APPROVED** [review]
No comments from me, I believe Danny's expertise :)

**Pull request merged**